pandas/io/pickle.py

```diff
@@ -62,7 +62,6 @@ def to_pickle(obj, path, compression='infer', protocol=pkl.HIGHEST_PROTOCOL):
     2    2    7
     3    3    8
     4    4    9
-
```

Comment:
can you add back the blank lines you removed

Reply:
added back.